lib/dice/src/mfg.rs

```diff
@@ -299,6 +303,74 @@ impl DiceMfg for SerialMfg<'_> {
                         syscon_locked,
                     })
                 }
+                MfgMessage::GetKeySlotStatus => {
+                    /// Layout of the Customer Field Programmble Area structure in Flash.
+                    ///
+                    /// This struct should be exactly 512 bytes. If you change it such that its size
+                    /// is no longer 512 bytes, it will not compromise security, but bootleby will
+                    /// panic while checking the persistent settings (and with any luck the static
+                    /// assertion below will fire before you hit the panic).
+                    #[derive(FromBytes)]
+                    #[repr(C)]
+                    struct CfpaPage {
+                        // Fields defined by NXP:
+                        header: u32,
+                        monotonic_version: u32,
+                        _fields_we_do_not_use: [u32; 4],
+                        rotkh_revoke: u32,
+                        _more_fields_we_do_not_use: [u32; 5],
+                        _prince_ivs: [[u32; 14]; 3],
+                        _nxp_reserved: [u32; 10],
+                        _customer_area: [u8; 224],
+                        _digest: [u8; 32],
+                    }
```

Comment:
ugh I really wish we had this accessible in another crate but I don't have a better idea at  the moment.